sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -530,48 +653,51 @@ abstract class AbstractSync(
      * @param issueInfo Issue to sync
      * @param label Label to sync
```

Comment:
docs for added params

Reply:
fafafc8c6b5332803aa70add51f61e003561ff7d